src/main/java/com/axibase/tsd/client/HttpClientManager.java

```diff
@@ -37,7 +39,7 @@
     private ClientConfiguration clientConfiguration;
     private GenericObjectPoolConfig objectPoolConfig;
 
-    private AtomicReference<GenericObjectPool<HttpClient>> objectPoolAtomicReference = new AtomicReference<GenericObjectPool<HttpClient>>();
+    private AtomicReference<GenericObjectPool<HttpClient>> objectPoolAtomicReferenc = new AtomicReference<GenericObjectPool<HttpClient>>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.shmagrinsky.com/coding_rules#rule_key=squid%3AS2293)